### PR TITLE
Voronoi

### PR DESCRIPTION
**Pull request by ryanbateman**
myoutput

Generate and show a Voronoi diagram.

So. Many. Hax.

### CODE REVIEW

File: mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/HypercatPainter.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        }
+
+        GraphEdge edge = graphEdges.get(graphEdgeCounter);
+        canvas.drawLine((float) edge.x1, (float) edge.y1, (float) edge.x2, (float) edge.y2, paint);
+        if (graphEdgeCounter < graphEdges.size()) {
+            graphEdgeCounter += 1;
+            try {
+                Thread.sleep(100);
+            } catch (Exception e) {
+
```
Comment (ryanbateman) [inline mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/HypercatPainter.java]: haha

File: mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/HypercatPaintingEngine.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import com.novoda.canvas.drawingmachine.BaseEngine;
+import com.novoda.canvas.drawingmachine.DrawingMachineView;
+
+class HypercatPaintingEngine extends BaseEngine {
+
+    private static final long PERIOD_PAINT_MILLIS = 17;
+    private DrawingMachineView drawingMachineView;
+    private final HypercatPainter painter;
+
+    public static HypercatPaintingEngine newInstance() {
```
Comment (ryanbateman) [inline mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/HypercatPaintingEngine.java]: i don't know why it's called hypercat, i just don't

File: mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/HypercatPainter.java (hunk 1 above)
Comment (ronocod) [inline mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/HypercatPainter.java]: Impeccable error handling

**Comment by blundell** [comment]
neat!

File: mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/Point.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/*
+  Copyright 2011 James Humphreys. All rights reserved.
```
Comment (blundell) [inline mobile/src/androidTest/java/com/novoda/canvas/drawingmachine/hypercat/Point.java:2]: related to kelly?

**Pull request merged**